website/docs/reference/model-properties.md

```diff
@@ -16,7 +16,7 @@ models:
     [description](description): <markdown_string>
     [docs](/reference/resource-configs/docs):
       show: true | false
-    [latest_version](resource-properties/latest-version): <version_identifier>
+    [latest_version](resource-properties/latest_version): <version_identifier>
```

Comment:
i dont understand why an underscore was added here, the actual page is https://docs.getdbt.com/reference/resource-properties/latest-version and a `latest_version` brings the user to a 'page not found'. suggesting it goes back to the `latest-version`

```suggestion
    [latest_version](resource-properties/latest-version): <version_identifier>
```

Reply:
@mirnawong1 Oops! I meant to rename the page from `latest-version` to `latest_version`.

The name of this resource property is `latest_version` (underscore). I looked at some other similar properties/configs, and they all have underscores in their file name / `id`:
- https://docs.getdbt.com/reference/resource-configs/sql_header
- https://docs.getdbt.com/reference/resource-configs/column_types
- https://docs.getdbt.com/reference/resource-configs/store_failures
(etc)